plugins/org.wso2.developerstudio.eclipse.gmf.esb.persistence/src/org/wso2/developerstudio/eclipse/gmf/esb/internal/persistence/FailoverEndPointTransformer.java

```diff
@@ -132,6 +132,8 @@ public FailoverEndpoint create(TransformationInfo info, FailoverEndPoint visualE
 			synapseFailEP.setName(name);
 		}
 
+		synapseFailEP.setBuildMessageAtt(visualEndPoint.isBuildMessage());
+
```

Comment:
Remove those unnecessary empty lines from this PR.

Reply:
Done